config/assets/default.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
       'modules/*/client/*.js',
       'modules/*/client/**/*.js'
     ],
-    views: ['modules/*/client/views/**/*.html'],
+    views: ['modules/*/client/**/*client.view.html'],
```

Comment:
I'm not sure what the best practice in the cases of where to put the html templates for directives.. but I usually put them in my *views* folder. Perhaps, someone else has some insight into that. But generally, I'd be weary of making changes to a core feature like the assets locations.